communication/slack-config/channels.yaml

```
@@ -34,6 +34,7 @@ channels:
   - name: chartcenter
   - name: chartmuseum
   - name: charts
+  - name: chile
```

Based on the convention of naming the channels after the official country code, how about:
```suggestion
  - name: cl-events
  - name: cl-users
```
?